Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -541,6 +541,15 @@
         "Icon": "/Assets/Helpers.png",
         "DocumentationUrl": "https://raw.githubusercontent.com/Microsoft/UWPCommunityToolkit/master/docs/helpers/BackgroundTaskHelper.md"
       },
+	  {
+        "About": "AppPinManager help to eaily add the app shortcut in StartMenu or TaskBar",
```

Comment:
Please fix wording, should be "The AppPinManager helps easily add app shortcuts to the Start Menu or the Taskbar"

Reply:
Done